[16.0][IMP] stock_release_channel: Set default state to asleep and add new criteria to disable pickings from being automatically assigned

tuantrantg opened the pull request.

Commit:
Add stock_available_to_promise_release

Commit:
[MIG] stock_available_to_promise_release: Migration to 13.0

oca-travis committed:
[UPD] Update sale_stock_available_to_promise_release.pot

OCA-git-bot committed:
[ADD] icon.png

OCA-git-bot committed:
[UPD] README.rst

Commit:
[IMP] various improvements to stock_available_to_promise_release

oca-travis committed:
[UPD] Update sale_stock_available_to_promise_release.pot

OCA-git-bot committed:
sale_stock_available_to_promise_release 13.0.1.2.0

Commit:
Add smart button on sales orders to show move allocations

Commit:
Add details on fields/filters of Transfers Allocations view

Commit:
Move 'sales_team' group to module depending on sale

OCA-git-bot committed:
sale_stock_available_to_promise_release 13.0.1.3.0

sebastienbeau committed:
[IMP] set module uninstallable

sebalix committed:
[FIX] 'Release ready' based on the shipping policy of the order

Commit:
Change license of stock_available_to_promise_release to LGPL-3

hparfr committed:
fix missing url key in manifests

mmequignon committed:
13.0: s_s_available_to_promise_release: Sale report product line availability

mmequignon committed:
13.0: s_s_available_to_promise_release: report fixes, translations

sebalix committed:
[MIG] sale_stock_available_to_promise_release: Migration to 14.0

sebalix committed:
[FIX] sale_stock_available_to_promise_release: typo

sebalix committed:
[IMP] sale_stock_available_to_promise_release: put availability data computation in its own method

oca-travis committed:
[UPD] Update sale_stock_available_to_promise_release.pot

OCA-git-bot committed:
[UPD] README.rst

ibuioli committed:
Added translation using Weblate (Spanish (Argentina))

ibuioli committed:
Translated using Weblate (Spanish (Argentina))

Currently translated at 100.0% (29 of 29 strings)

Translation: wms-14.0/wms-14.0-sale_stock_available_to_promise_release
Translate-URL: https://translation.odoo-community.org/projects/wms-14-0/wms-14-0-sale_stock_available_to_promise_release/es_AR/

sebalix committed:
[IMP] sale_stock_available_to_promise_release_dropshipping: add test

simahawk committed:
sale_stock_available_to_promise_release: dev status -> Beta

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
sale_stock_available_to_promise_release 14.0.1.1.0

sebalix committed:
Fix access right issues when using actions

OCA-git-bot committed:
sale_stock_available_to_promise_release 14.0.1.1.1

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 69.2% (18 of 26 strings)

Translation: wms-14.0/wms-14.0-sale_stock_available_to_promise_release
Translate-URL: https://translation.odoo-community.org/projects/wms-14-0/wms-14-0-sale_stock_available_to_promise_release/it/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 69.2% (18 of 26 strings)

Translation: wms-14.0/wms-14.0-sale_stock_available_to_promise_release
Translate-URL: https://translation.odoo-community.org/projects/wms-14-0/wms-14-0-sale_stock_available_to_promise_release/it/

sbejaoui committed:
[ADD] - add stock_release_channel_geoengine

This module enhance release channels with the addition of
a delivery zone that can be selected by the delivery manager through the UI.
If a zone is specified, the release channel will exclusively select deliveries
for partners who are localized within that zone.

mymage committed:
Translated using Weblate (Italian)

Currently translated at 69.2% (18 of 26 strings)

Translation: wms-14.0/wms-14.0-sale_stock_available_to_promise_release
Translate-URL: https://translation.odoo-community.org/projects/wms-14-0/wms-14-0-sale_stock_available_to_promise_release/it/

glitchov committed:
[16.0][ADD] stock_release_channel_delivery

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 69.2% (18 of 26 strings)

Translation: wms-14.0/wms-14.0-sale_stock_available_to_promise_release
Translate-URL: https://translation.odoo-community.org/projects/wms-14-0/wms-14-0-sale_stock_available_to_promise_release/it/

jbaudoux committed:
[ADD] stock_release_channel_plan

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 69.2% (18 of 26 strings)

Translation: wms-14.0/wms-14.0-sale_stock_available_to_promise_release
Translate-URL: https://translation.odoo-community.org/projects/wms-14-0/wms-14-0-sale_stock_available_to_promise_release/it/

OCA-git-bot committed:
[UPD] README.rst

sbejaoui committed:
[IMP] - stock_release_channel_delivery: better handle carrier filter

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (26 of 26 strings)

Translation: wms-14.0/wms-14.0-sale_stock_available_to_promise_release
Translate-URL: https://translation.odoo-community.org/projects/wms-14-0/wms-14-0-sale_stock_available_to_promise_release/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (26 of 26 strings)

Translation: wms-14.0/wms-14.0-sale_stock_available_to_promise_release
Translate-URL: https://translation.odoo-community.org/projects/wms-14-0/wms-14-0-sale_stock_available_to_promise_release/it/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: wms-14.0/wms-14.0-sale_stock_available_to_promise_release
Translate-URL: https://translation.odoo-community.org/projects/wms-14-0/wms-14-0-sale_stock_available_to_promise_release/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (29 of 29 strings)

Translation: wms-14.0/wms-14.0-sale_stock_available_to_promise_release
Translate-URL: https://translation.odoo-community.org/projects/wms-14-0/wms-14-0-sale_stock_available_to_promise_release/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (29 of 29 strings)

Translation: wms-14.0/wms-14.0-sale_stock_available_to_promise_release
Translate-URL: https://translation.odoo-community.org/projects/wms-14-0/wms-14-0-sale_stock_available_to_promise_release/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (29 of 29 strings)

Translation: wms-14.0/wms-14.0-sale_stock_available_to_promise_release
Translate-URL: https://translation.odoo-community.org/projects/wms-14-0/wms-14-0-sale_stock_available_to_promise_release/it/

mt-software-de committed:
[IMP] stock_available_to_promise_release: Own field for release policy

Currently if sale_stock_available_to_promise_release is installed
we are not able to change the release policy anymore.
Therefor we need a separate field,
which will be set from the procurement.group's move_type.

For Example we can now keep the shipping_policy on the transfer
but release only partially.

grindtildeath committed:
[IMP] stock_release_channel_plan: weekdays

Add preparation weekdays to stock.release.channel

lmignon committed:
[FIX] stock_release_channel_delivery: No assignation if carriers differ

A picking without carrier should not be assigned to a release channel restricted to a specific carrier.

lmignon committed:
[IMP] stock_release_channel_delivery: Add missing contributors

lmignon committed:
Update stock_release_channel_delivery/models/stock_release_channel.py

Co-authored-by: Jacques-Etienne Baudoux <je@bcim.be>

trisdoan committed:
[IMP] stock_release_channel_delivery: allow to configure multiple shipping methods on a channel

florentx committed:
[IMP] sale_stock_available_to_promise_release: Add delayed_qty on SO line

Commit:
[IMP] sale_stock_available_to_promise_release: pre-commit stuff

Commit:
[MIG] sale_stock_available_to_promise_release: Migration to 16.0

sebalix committed:
sale_stock_available_to_promise_release: improve '_get_next_replenishment_date' perf

Checking ongoing states instead of finished ones (done and canceled) helps
PostgreSQL to perform the filter on far less lines (done lines are
increasing a lot over time), speeding up the query to get the next
replenishment date.

jbaudoux committed:
[FIX] stock_release_channel_propagate_channel_picking

Fix failing test when more channels are loaded

nilshamerlinck reviewed (changes requested):
> Set default to release channel and add criteria to disable picking automatically assigned to the channel

Set default state to asleep and add new criteria to disable pickings from being automatically assigned

OCA-git-bot committed:
Merge PR #599 into 16.0

Signed-off-by jbaudoux

tuantrantg commented:
hi @nilshamerlinck please review it again

Commit:
[UPD] Update stock_release_channel_geoengine.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #840 into 16.0

Signed-off-by jbaudoux

Commit:
[UPD] Update stock_release_channel_delivery.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: wms-16.0/wms-16.0-stock_release_channel_delivery
Translate-URL: https://translation.odoo-community.org/projects/wms-16-0/wms-16-0-stock_release_channel_delivery/

OCA-git-bot committed:
Merge PR #874 into 16.0

Signed-off-by simahawk

Commit:
[UPD] Update sale_stock_available_to_promise_release.pot

OCA-git-bot committed:
[BOT] post-merge updates

jbaudoux committed:
fixup! [ADD] stock_release_channel_plan

jbaudoux committed:
[FIX] stock_release_channel_shipment_lead_time

Fix failing tests when demo channels are already asleep

OCA-git-bot committed:
Merge PR #695 into 16.0

Signed-off-by jbaudoux

Commit:
[UPD] Update stock_release_channel_plan.pot

OCA-git-bot committed:
[BOT] post-merge updates

tuantrantg committed:
[IMP] stock_release_channel: Set default state to asleep

tuantrantg committed:
[IMP] stock_release_channel: Add new criteria to disable pickings from being automatically assigned
